app/views/admin/projects.html.erb

```diff
@@ -31,12 +31,14 @@ See doc/COPYRIGHT.rdoc for more details.
   <%= render :partial => 'no_data' if @no_configuration_data %>
 </div>
 
-<% content_for :action_menu_specific do %>
-  <%= link_to l(:label_project_new), {:controller => '/projects', :action => 'new'}, :class => 'icon icon-add' %>
-<% end %>
 <% html_title(l(:label_administration), l(:label_project_plural)) -%>
-<h2><%=l(:label_project_plural)%></h2>
-<%= render :partial => 'layouts/action_menu_specific' %>
+<%= toolbar title: l(:label_project_plural) do %>
+  <li class="toolbar-item">
+    <%= link_to new_project_path, class: 'button -alt-highlight' do %>
+      <i class="button--icon icon-add"></i> <%= l(:label_project_new) %>
```

Comment:
@floriank If you use decide to include an `<i>` element within the button content, then you should also wrap the text in a span with `.button--text` applied, e.g.:
```erb
<span class="button--text"><%= l(:label_project_new) %></span>
```

I know the semantics are a bit confusing. The above was optimised for a _composite button_ use case, where there is more than one icon used within the button, or an icon + text+ dropdown indicator.

An alternative option, which should serve most use cases, is to apply `button -alt-highlight -with-icon icon-add` to the ` <button>` or `<a>` element.